be/src/runtime/exec_env.cpp

```diff
@@ -216,7 +219,7 @@ Status ExecEnv::_init(const std::vector<StorePath>& store_paths) {
 #ifndef USE_STAROS
     _lake_group_assigner = new FixedGroupAssigner(_store_paths.front().path);
```

Comment:
why _stores_paths.front()?

Reply:
`FixedGroupAssigner` is not for production environments, any local path would be ok.